molo/commenting/admin_import_export.py

```diff
@@ -10,17 +10,26 @@ class MoloCommentsResource(resources.ModelResource):
     article_subtitle = Field()
     article_full_url = Field()
     parent_id = Field()
+    country = Field()
 
     class Meta:
         model = MoloComment
 
-        exclude = ('comment_ptr', 'content_type', 'object_pk',
-                   'site', 'user', 'user_url', 'lft', 'rght',
+        exclude = ('site', 'comment_ptr', 'content_type', 'object_pk',
+                   'user', 'user_url', 'lft', 'rght',
                    'tree_id', 'level', 'ip_address', )
 
-        export_order = ('submit_date', 'user_name', 'user_email', 'comment',
-                        'id', 'parent_id', 'article_title', 'article_subtitle',
-                        'article_full_url', 'is_public', 'is_removed')
+        export_order = (
+            'country', 'submit_date', 'user_name', 'user_email', 'comment',
+            'id', 'parent_id', 'article_title', 'article_subtitle',
+            'article_full_url', 'is_public', 'is_removed')
+
+    def dehydrate_country(self, comment):
+        if not comment.content_object or not \
+                hasattr(comment.content_object, 'subtitle'):
+            return ''
```

Comment:
I don't think you need this conditional at the top of the `dehydrate` - assuming there should always be a site with a root page and a title?